examples/example.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
This is ok but by using `index.html` you can get a shorter demo url, like https://publiclab.github.io/leaflet-blurred-location/examples/